operations/deployment/terraform/security-group.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_security_group" "ec2_security_group" {
-  name        = "${var.aws_resource_identifier}-instance-sg"
+  name        = var.security_group_name
```

Comment:
This is a breaking-locking change. If you run the action with just a new version, the name-change will make the action run forever, as it can't recreate the sg (locked because being used)
That's why I reverted this